Library/Homebrew/extend/os/mac/software_spec.rb

```diff
@@ -4,6 +4,9 @@
 class SoftwareSpec
   undef uses_from_macos
 
+  sig { params(spec: T.any(String, T::Hash[T.untyped, T.untyped])).void }
+  def remove_gnu_compilers(spec); end
```

Comment:
I'd suggest defining this in `Library/Homebrew/software_spec.rb` instead and omitting it here.

Reply:
Done